src/plugins/vis_types/timeseries/public/application/components/aggs/field_select/field_select.tsx

```diff
@@ -47,10 +47,19 @@ interface FieldSelectProps {
   fullWidth?: boolean;
 }
 
-const getPreselectedFields = (
+const getPlaceholderValue = (
   placeholder?: string,
   options?: Array<EuiComboBoxOptionOption<string>>
-) => placeholder && findInGroupedOptions(options, placeholder)?.label;
+) => {
+  let returnedValue = placeholder;
+  if (placeholder) {
+    const label = findInGroupedOptions(options, placeholder)?.label;
+    if (label) {
+      returnedValue = label;
+    }
+  }
+  return returnedValue;
```

Comment:
nit: just a shorter version:

```suggestion
if(!placeholder){
  return;
}
return findInGroupedOptions(options, placeholder)?.label || placeholder;
```